wiki/Help_Centre/de.md

```diff
@@ -0,0 +1,79 @@
+---
+tags:
+  - help
+  - issue
+  - problem
+  - trouble
+  - missing
+  - Hilfe
+  - Fehler
+  - Problem
+---
+
+# Hilfecenter
+
+Hast du bei irgendetwas Probleme? Wir sind hier, um zu helfen! Finde einige Lösungen zu alltäglichen Problemen in der Sitebar deines Bildschirms. Wenn dein Problem hier nicht auftaucht oder du immer noch keine Lösung hast, dann sende eine E-Mail zu [support@ppy.sh](mailto:support@ppy.sh). Alternativ kannst du dein Problem auch im [Hilfeforum](https://osu.ppy.sh/community/forums/5) schildern, in dem dir jemand weiterhilft.
+
+## Abschnitte
+
+Wähle den Abschnitt, der deinem Problem am meisten ähnelt, damit du eine passende Lösung findest.
+
+| Abschnitt | Schwerpunkte |
+| :-- | :-- |
+| [Account](/wiki/Help_Centre/Account) | osu!supporter, Einschränkungen, Einloggen, Namensänderungen, Profildaten |
+| [Mapping und Editor](/wiki/Help_Centre/Beatmapping) | Deine Beatmaps pflegen, Beatmap-Eigentümerschaft, Beatmap-Slots |
+| [Client](/wiki/Help_Centre/Client) | Fehler und Abstürze, Spielverlauf, Verbindung, Leistung |
+| [Installation dnd Registrierung](/wiki/Help_Centre/Installation_and_registration) | Das Spiel herunterladen, einen Account erstellen |
+| [Store](/wiki/Help_Centre/Store) | Merchandise, osu!go, osu!keyboard, osu!tablet |
+| [Webseite](/wiki/Help_Centre/Website) | Benutzer blockieren, den Support kontaktieren, Seitengestaltung |
+
+## Helfe uns, dir zu helfen
+
+### Log-Dateien
+
+**Log-Dateien sind genaue Aufzeichnungen was der Spiel-Client zu einem bestimmten Zeitpunkt gemacht hat. Wir können sie verwenden, um dein Problem zu lösen.**
+
+Diese Dateien sind extrem praktisch und können sogar das Lösen komplexer Probleme trivial machen.
+
+Wenn ein Mitglied des Supportteams dich nach diesen Logs gefragt hat, dann mache Folgendes:
+
+1. Öffne osu!.
+2. Klicke den `Optionen`-Knopf im Hauptmenü oder drücke `Strg` + `O`.
+3. Tippe `Updatequelle` in die Schnellsuchleiste. Das wird dich zu der aktuellen Updatequelle deines Clients bringen.
+4. Versichere dich, dass es auf `Cutting Edge (Experimentell)` eingestellt ist.
+5. Klicke den Knopf zum Neustarten, der unten auf dem Bildschirm erscheint, sobald du die Updatequelle änderst.
+6. Gehe nochmal zum `Optionen`-Menü und klicke `osu!-Ordner öffnen`.
+7. Finde das `Logs`-Verzeichnis in dem Fenster, welches sich öffnet.
+8. Wähle die benötigte Log-Datei (der Support-Vertreter wird dir sagen, welche) und hänge sie an das Support-Ticket oder den Forum-Beitrag.
```

Comment:
```suggestion
8. Wähle die benötigte Log-Datei (der Support-Ansprechpartner wird dir sagen, welche) und hänge sie an das Support-Ticket oder den Forum-Beitrag an.
```

Reply:
I think `an den Forum-Beitrag.` sounds a bit bette. What do you think?